### ISSUE TITLE
[TASK] Change Lighthouse configuration

### ISSUE THREAD

**Issue by MagdalenaLarge** [ONSdigital/design-team#221]
**Description**

Currently we are getting a warning in Lighthouse while doing the testing, but we do not get error message.
It means that errors are quite often missed.

**Action**
Investigate what is causing lack of error messages in the Lighthouse and fix it so any future issues are highlighted in error message

Ticket depends on completion of the tickets #215 , #216 , #217 , #218 , #219 .

Once you have finished with all above components change the below code from the lighthouse/lighthouserc.js :

**OLD**
```
assert: {
    assertions: {
        'categories:accessibility': ['warn', { minScore: 1 }],
    },
},
```

**NEW**
```
assert: {
    assertions: {
        'categories:accessibility': ['error', { minScore: 1 }],
    },
},
```

**Comment by rmccar** [comment]
Blocked waiting for other PRs to be merged

**Comment by rmccar** [comment]
The scope of this ticket will need to be extended to look into ignoring the issues we have with the setting of the `aria-expanded` attribute on radios. This is covered in #216

**Comment by rmccar** [comment]
Blocked waiting on outcome of #216
